package.json

```diff
@@ -10,11 +10,11 @@
     "prepublish": "npm run build",
     "lint": "eslint --ext .js,.jsx src test stories",
     "storybook": "start-storybook -p 9001",
-    "pretest": "npm run lint",
-    "test": "npm run build && npm run test:coverage",
-    "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- -R tap 'test/**/*-test.*'",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/*-test.*",
+    "pretest": "npm run build && npm run lint",
+    "test": "nyc -e .jsx --reporter=lcov --reporter=text npm run test:mocha",
+    "tests-only": "npm run build && npm run test:mocha",
+    "test:mocha": "SKIP_DOM=1 npm run test:_mocha && npm run test:_mocha -- --require './test/.dom.js'",
```

Comment:
i think the "without dom" and "with dom" test runs should be their own standalone commands.